tmt/steps/finish/__init__.py

```diff
@@ -50,6 +51,15 @@ def finish(context: 'tmt.cli.Context', **kwargs: Any) -> None:
 
         return finish
 
+    @classmethod
+    def options(cls, how: Optional[str] = None) -> List[tmt.options.ClickOptionDecoratorType]:
+        return [
+            click.option(
+                '-k', '--keep', is_flag=True,
+                help='Keep all test files rsynced to local host (Skip pruning).'
```

Comment:
Originally I thought the proper place for the new `--keep` option would be the `run` command as it is similar to what `--remove` does there:

    Options:
    -i, --id ID                     Run id (name or directory path).
    -l, --last                      Execute the last run once again.
    -r, --rm, --remove              Remove the workdir when test run is finished.

But here it also make sense. @happz, @lukaszachy, any recommendations here?

Reply:
Personally I'd prefer to have it as `run` option, otherwise I would need to add finish step on the commandline most of the times.

Reply:
I've been modifying some tests analyzing created files, and adding this parameter to them. And yes, it is annoying to write `finish` everywhere. Thus, I moved it under `run` as you suggested.  It is more convenient.

Reply:
Yes, good point, under `run` it will be more handy.